src/resources/views/crud/fields/select2_from_ajax.blade.php

```diff
@@ -163,23 +170,37 @@ function bpFieldInitSelect2FromAjaxElement(element) {
                 processResults: function (data, params) {
                     params.page = params.page || 1;
 
-                    var result = {
-                        results: $.map(data.data, function (item) {
-                            textField = $fieldAttribute;
-                            return {
-                                text: item[textField],
-                                id: item[$connectedEntityKeyName]
+                    if(data.data) {
```

Comment:
@pxpm when do we may have `data.data`?

Would it be better to have something like;
```js
if(data.data) {
  data = data.data;
}
// or
data = data.data || data;
```

That way we would avoid repeating the code, and we would be encapsulating that hack (not sure if it's some sort of hack or not).

Reply:
I've just submitted a [commit](https://github.com/Laravel-Backpack/CRUD/pull/3439/commits/042ae6f627beaf33cdb1def3e81732c6092a463d) that addresses this. You are right at that. I am not sure it's hacky, it's just the way we separate the the paginated from non paginated results to define the `pagination: more` configuration later. 

Let me know what you think,
Pedro